test/integration/containerized/test_cli_containerized.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+import os
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def test_module_run(cli):
+    cli(['-m', 'ping','--hosts', 'localhost', 'run', os.path.join(HERE, 'priv_data')])
+
+
+def test_playbook_run(cli):
+    cli(['run', os.path.join(HERE,'priv_data'), '-p', 'test-container.yml'])
```

Comment:
:bulb: I want to add a test that runs a playbook that fails and confirms we get non-zero exit code

Reply:
@kdelee - good question - I poked around some existing tests and found [this one](https://github.com/jladdjr/ansible-runner/blob/containerized_playbook_runs/test/integration/test_events.py#L14) that already asserts that the run was successful. It also does a sanity check on several events that we expect to see. What do you think about parameterizing that to run w/ and w/out containerization?

I think some of the other tests in that module may also be good candidates for parameterization if we think that running inside a container puts those other aspects of a run at risk. The `test_basic_events` test may give us the most bang for our buck, though.

Reply:
got `test_basic_events` to work w/ containerization - actually helped me work out some kinks / understand how some things were connected:
https://github.com/jladdjr/ansible-runner/blob/containerized_playbook_runs/test/integration/test_events.py#L14-L55